src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -185,23 +221,25 @@ const NavHeader = (props) => {
           />
         </View>
       )}
-    </Animated.View>
+    </View>
   );
 };
 
 NavHeader.propTypes = {
+  initialData: PropTypes.object,
   onShareCommunity: PropTypes.func,
   isHeaderHide: PropTypes.bool,
   isFollow: PropTypes.bool,
   hideSeeMember: PropTypes.bool,
   animatedHeight: PropTypes.number,
   opacityHeaderAnimation: PropTypes.number,
+  opacityImage: PropTypes.number,
   getTopicDetail: PropTypes.func,
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
   setIsFollow: PropTypes.func,
-
+  isLoading: PropTypes.bool,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `initialData` and `isLoading` props should be properly handled in the parent components that use `NavHeader`. Make sure that these props are always passed when necessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->